src/workflows/import-docmaps.ts

```diff
@@ -41,6 +41,7 @@ export async function importDocmaps(docMapIndexUrl: string, s3StateFileUrl?: str
     parentClosePolicy: ParentClosePolicy.PARENT_CLOSE_POLICY_ABANDON,
     // makes sure there is only one workflow running, this new one.
     workflowIdReusePolicy: WorkflowIdReusePolicy.WORKFLOW_ID_REUSE_POLICY_TERMINATE_IF_RUNNING,
+    taskQueue: 'import-docmaps',
```

Comment:
@scottaubrey: don't we need to read this value from env variable? like: `taskQueue: env.process.TASK_QUEUE_NAME`

Reply:
We probably could do, but maybe as a follow up